enforcer/datapath.go

```diff
@@ -20,23 +19,6 @@ import (
 	"github.com/aporeto-inc/trireme/policy"
 )
 
-// InterfaceStats for interface
```

Comment:
When we actually do it, we should do it carefully. There are race conditions because of the stats, and I don't want to add locks all over the place for that.